### ISSUE TITLE
[unity] Cloned MeshAttachment when repacking skin does not clone color

### ISSUE THREAD

**Issue by DanniSchou** [EsotericSoftware/spine-runtimes#1187]
Using skin.GetRepackedSkin(...) with MeshAttachments results in the attachment being cloned with white regardless of the actual color.
The issue is located [here](https://github.com/EsotericSoftware/spine-runtimes/blob/3.6/spine-unity/Assets/spine-unity/Modules/AttachmentTools/AttachmentTools.cs#L1047).
```
mesh.r = 1f;
mesh.g = 1f;
mesh.b = 1f;
mesh.a = 1f;
```
Should be changed to:
```
mesh.r = o.r;
mesh.g = o.g;
mesh.b = o.b;
mesh.a = o.a;
```

**Comment by pharan** [comment]
hmm. thanks for reporting this!

**Comment by pharan** [comment]
This has been applied to 3.7-beta. https://github.com/EsotericSoftware/spine-runtimes/commit/c81f9dbe62eb0ba0ab4336324a3b08ac5b540164
Are you using 3.6? It should be easy to backport.

**Comment by DanniSchou** [comment]
Thanks, that was super fast!
I'm currently on 3.6. I've added the fix locally since I'm using the unitypackage.

**Comment by pharan** [comment]
The fix was applied to 3.6, including the unitypackage.
Thanks again for reporting this problem!
